cache-caffeine/src/main/java/io/micronaut/cache/caffeine/DefaultSyncCache.java

```diff
@@ -189,7 +187,7 @@ public void put(@NonNull Object key, @Nullable Object value) {
      * @param cacheConfiguration The cache configurations
      * @return cache
      */
-    protected com.github.benmanes.caffeine.cache.Cache buildCache(CacheConfiguration cacheConfiguration) {
+    protected com.github.benmanes.caffeine.cache.Cache buildCache(CaffeineCacheConfiguration cacheConfiguration) {
```

Comment:
This breaks binary compatibility, you would need to revert this change and introduce an instanceof check

Reply:
Done